eq-author/src/App/page/Design/index.js

```diff
@@ -82,7 +94,25 @@ export const UnwrappedPageRoute = (props) => {
       return <Loading height="38rem">Page loading…</Loading>;
     }
 
-    return <Error>Something went wrong</Error>;
+    if (error) {
+      return <Error>Something went wrong</Error>;
+    }
+  };
+
+  const redirectPage = () => {
+    //need this questionnaire != null otherwise TypeError when recompile and run author.
+    if (isEmpty(page) && questionnaire !== null) {
+      return (
+        <RedirectRoute
+          from={"/q/:questionnaireId/page/:pageId/design"}
+          to={
+            "/q/:questionnaireId/page/" +
+            questionnaire?.sections[0]?.folders[0]?.pages[0]?.id +
+            "/design"
+          }
+        />
+      );
+    }
```

Comment:
```suggestion
    if (isEmpty(page)) {
      return (
        <RedirectRoute
          from={"/q/:questionnaireId/page/:pageId/design"}
          to={
            "/q/:questionnaireId/page/" +
            questionnaire?.sections[0]?.folders[0]?.pages[0]?.id +
            "/design"
          }
        />
      );
    }
```

The ?s will handle null conditions